book/07-git-tools/sections/rerere.asc

```diff
@@ -249,4 +249,4 @@ $ git rebase --continue
 Applying: i18n one word
 ----
 
-So, if you do a lot of re-merges, or want to keep a topic branch up to date with your master branch without a ton of merges, or you rebase often, you can turn on `rerere` to help your life out a bit.
+Kaya, kung ikaw ay maraming gagawin na mga pag-re-remerge, o nais na panatilihin ang isang paksa na branch na bagongbago sa maraming mga merge, o madalas mong i-rebase, maaari mong i-on ang `rerere` upang tulungan ang iyong bugay nang kaunti.
```

Comment:
* `re-merges` could be translated to `pag-remerge`
* `bagongbago` to `bagong-bago`
* `bugay` to `buhay`

Reply:
done change